exec/bin/burnio/burnio.go

```diff
@@ -145,7 +145,7 @@ func burnWrite(directory, size string) {
 func burnRead(directory, size string) {
 	// create a 1g file under the directory
 	tmpFileForRead := path.Join(directory, readFile)
-	createArgs := fmt.Sprintf("if=/dev/zero of=%s bs=%dM count=%d oflag=dsync", tmpFileForRead, 6, count)
+	createArgs := fmt.Sprintf("if=/dev/zero of=%s bs=%sM count=%d oflag=dsync", tmpFileForRead, size, count)
```

Comment:
Why did you modify `6` to size variable?

Reply:
why use a constant here? if the total disk  size is less than 600M，this experiment will fail.

Reply:
The purpose of creating a file here is to provide a readable file, so the file size can be as long as it is greater than the maximum read speed of the disk. No need to follow size changes. If the available disk size is less than 600M, it's a problem. I accept your changes first, we will open an issue later to discuss how to modify this place.